pylint_django/transforms/__init__.py

```diff
@@ -1,13 +1,20 @@
-"""Transforms."""
+"""
+These transforms replace the Django types with adapted versions to provide
+additional typing and method inference to pylint. All of these transforms
+are considered "global" to pylint-django, in that all checks and improvements
+requre them to be loaded. Additional transforms specific to checkers are loaded
+by the checker rather than here.
+
+For example, the ForeignKeyStringsChecker loads the foreignkey.py transforms
+itself as it may be disabled independently of the rest of pylint-django
+"""
 import os
 import re
 
 import astroid
 
-from pylint_django.transforms import foreignkey, fields
+from pylint_django.transforms import fields
 
-
-foreignkey.add_transform(astroid.MANAGER)
```

Comment:
Could benefit from a comment either here or on the doc-string text and/or the foreignkey module to explicitly state why `foreignkey.add_transform()` isn't called like for the other modules.

Reply:
There is a module-level docstring explaining this already which I think is sufficient: https://github.com/PyCQA/pylint-django/blob/feature/improve-django-settings-handling/pylint_django/transforms/__init__.py#L1

Reply:
I also added this issue as a discussion for moving all transforms/augmentations into a checker rather than being loaded immediately by the plugin: https://github.com/PyCQA/pylint-django/issues/290